### ISSUE TITLE
Unable to get basic relative module resolution to work

### ISSUE THREAD

**Issue by nzbart** [facebook/flow#4942]
I'm struggling with something that appears to be so basic that I'm sure I'm doing something wrong. I've searched for open issues here with the tag "module resolution", and can't find anything relevant.

I'm unable to get basic module resolution of the following pattern to work:
```javascript
import Foo from './foo'
```

I have created a tiny example based on the official Flow documentation by doing the following:
1. Creating a new directory.
2. Running `flow init`.
3. Copying `export.js` and `import.js` from the [documentation on Module Types](https://flow.org/en/docs/types/modules/).
4. Running `flow check`.

Interestingly, I get the same error at the command-line as I see on the Module Types documentation page:
![image](https://user-images.githubusercontent.com/1592130/30716986-ba29d56c-9f6f-11e7-8f25-6c99d9a6f618.png)

Is it intentional that this error also exists on the documentation page?

I'm running flow 0.52.0 on Windows.

**Comment by mrkev** [comment]
Was this issue solved? I just noticed the error on the website, I don't think that was intentional.

**Comment by nzbart** [comment]
@mrkev I haven't retested since, but simply gave up on Flow because I couldn't get such a simple example to work. I assumed that it wasn't aimed at my use-case.

**Comment by mrkev** [comment]
Hmmm I just did the exact same on my computer and I get no issue. I don't have a Windows machine to test this in, but you're right, importing modules is not only simple but crucial-- not being able to do so is a pretty big bug that pretty much would break all usage of flow, so the fact there weren't/aren't any other reports of this being broken leads me to believe this was perhaps an issue with your environment/setup.

About the website-- I don't find it surprising that there's an error there. After all, there exist no `./exports.js` or `./imports.js` files on the server, this is all markdown, so while I wouldn't expect to see an error because it's javascript that's supposed to be there only for display, I'm not surprised there is one. I've documented this in a separate issue: #6607
